internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/server/TestAdminSparkServer.java

```diff
@@ -73,6 +73,7 @@
 import org.apache.http.entity.StringEntity;
 import org.apache.http.impl.nio.client.CloseableHttpAsyncClient;
 import org.apache.http.message.BasicNameValuePair;
+import org.apache.http.nio.client.HttpAsyncClient;
```

Comment:
why this?

Reply:
It's the base class for `CloseableHttpAsyncClient` and only used in a helper method. Let me know if you want me to remove it